docs/en/setup/service-agent/java-agent/README.md

```diff
@@ -87,6 +87,7 @@ Now, we have the following known optional plugins.
 * [Trace Oracle and Resin](agent-optional-plugins/Oracle-Resin-plugins.md)
 * [Filter traces through specified endpoint name patterns](agent-optional-plugins/trace-ignore-plugin.md)
 * Gson serialization lib in optional plugin folder
+* Lettuce 5.x(JRE1.8+) in optional plugin folder 
```

Comment:
Am I writing the correct version of Lettuce?

Reply:
I think so.